js/change-list.js

```diff
@@ -226,7 +247,7 @@ class ChangeList {
 
   applyChangeRange(mem8, mem32, start, len) {
     const end = (start + len) / 4;
-    for (let i = start / 4; i < end; ) {
+    for (let i = start / 4; i < end;) {
```

Comment:
```suggestion
    for (let i = start / 4; i < end; ) {
```

Nitpick: leave existing whitespace alone ;)